index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Notes Application</title>
+    <link rel="stylesheet" href="sanitize.css">
+    <link rel="stylesheet" href="style.css">
+    
+</head>
+<body>
+    <header>
+        <h1>My Notes Application</h1>
+    </header>
+    <main>
+        <section class="new-note">
+            <h2>Create a New Note</h2>
+            <label for="title">Title:</label>
```

Comment:
`for` attribute in `label` points to input `id`. 

Defining a value for attribute `for` is unuseful if `input` misses its `id` attribute.
```suggestion
            <label for="title">Title:</label>
            <input type="text" id="title" placeholder="Enter note title" required>
```